osu.Game/Skinning/SkinManager.cs

```diff
@@ -172,6 +175,8 @@ public void EnsureMutableSkin()
                     result.PerformRead(skin => Save(skin.CreateInstance(this)));
                     CurrentSkinInfo.Value = result;
                 }
+
+                return true;
```

Comment:
Shouldn't this be

```suggestion
                return result != null;
```

or similar? I'm not really even sure when the import flow can actually return a null `Live`, but the method is annotated as null-returning...

Reply:
I guess? Although at the point it's null I think the game would be in a very unhappy state (no idea what happens when `CurrentSkinInfo` is set to null..)